conda.recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,6 @@
+python:
+  - 2.7
+  - 3.5
+  - 3.6
+  - 3.7
+
```

Comment:
Looks like an extraneous empty line, could you please remove it?

Reply:
Okay, done.